### PR TITLE
Update Links to API Documentation

### PR DESCRIPTION
**Pull request by Ken-Lo**
* Change all links from https://developer.gocardless.com/pro/ to https://developer.gocardless.com/
* Redirect gocardless.com/features/api/ to https://developer.gocardless.com/ (page is outdated)
* Change link in footer to read 'API Documentation' instead of 'Pro API'

**TODO:**

- [x] Translate 'API Documentation' into Dutch, French, German, Portugese and Spanish (for link in footer)

### CODE REVIEW

**Comment by Ken-Lo** [comment]
Hey @jacobpgn 
Can I get a quick review before sending over to UX?
Thanks!

File: app/messages/es.js [hunk 1]
```diff
@@ -151,7 +151,7 @@ export default {
     description: '',
     nav_title: 'Equipo',
   },
-  pro_docs: {
+  api_docs: {
     title: 'Pro API',
```
Comment (jacobpgn) [inline app/messages/es.js]: - Pro

Comment (jacobpgn) [inline app/messages/es.js]: Remove `Pro`?

File: app/messages/fr.js [hunk 2]
```diff
@@ -274,7 +274,7 @@ export default {
     title: 'Paiements par prélèvement SEPA',
     description: 'Le prélèvement SEPA est plus fiable et économique que les cartes. C\'est également plus efficace que les virements et chèques. Apprenez-en plus sur les paiements par prélèvement.',
   },
-  pro_docs: {
+  api_docs: {
     title: 'Pro API',
```
Comment (jacobpgn) [inline app/messages/fr.js]: `Pro`

File: app/messages/nl.js [hunk 3]
```diff
@@ -72,7 +72,7 @@ export default {
     description: '',
     nav_title: 'Team',
   },
-  pro_docs: {
+  api_docs: {
```
Comment (jacobpgn) [inline app/messages/nl.js]: `P` `r` `o`

**Comment by jacobpgn** [comment]
Really nice 😸 Just some references to Pro that might need to go away?

File: app/messages/es.js (hunk 1 above)
Comment (Ken-Lo) [inline app/messages/es.js]: Was going to get API Documentation translated into each language, or could just go with `API` by itself?

Comment (jacobpgn) [inline app/messages/es.js]: That makes sense! Easy either way.

**Comment by Ken-Lo** [comment]
Hey @swillis 
Another small change to tidy up the footer :)
Any chance you'd be able to have a quick look-over?
Thanks!

**Comment by Ken-Lo** [comment]
@swillis Just a bump - would you be able to have a v quick look this week? Main thing is the change the footer & also the removal/redirect of the API splash page.

**Comment by Ken-Lo** [comment]
@swillis @jamesshedden Bump again, so we're linking to the right version of the API docs.

**Comment by Ken-Lo** [comment]
@swillis Just going to merge this in now - main thing for you guys to be aware of is we are deleting the gocardless.com/features/api/ splash page.

**Comment by jamesshedden** [comment]
@Ken-Lo Great, thanks. Apologies this one slipped thru the net - great work 👍

**Pull request merged**